src/components/Authentication/Login.vue

```diff
@@ -38,10 +38,11 @@
             :loading="inProgress")
           v-icon(left) $vuetify.icons.login
           | {{ otpFormVisible ? 'Verify code' : 'Login' }}
-        v-spacer
-        v-btn(
-            text, color="primary", :to="forgotPasswordRoute", :href="forgotPasswordUrl",
-            @click="$emit('forgotpassword')") Forgot Password?
+        template(v-if="showForgotPassword")
```

Comment:
I didn't realize until now that you could use `template` in this way.

Reply:
https://vuejs.org/v2/guide/conditional.html#Conditional-Groups-with-v-if-on-lt-template-gt

It's a good practice to avoid adding an unnecessary extra element level to the actual DOM.